producer-kafka/README.adoc

```diff
@@ -0,0 +1,407 @@
+= Using the Kafka CDC Producer
+
+This extension provides support for sending Change Data Capture(CDC) events to an Apache Kafka®.
+It is designed to work with the Stargate.
+
+== Kafka CDC - Stargate integration
+
+This extension is designed to integrate with Stargate. It uses an link:https://www.osgi.org/[OSGi] framework for registering the extension within Stargate.
```

Comment:
```suggestion
This extension is designed to integrate with Stargate. It uses a link:https://www.osgi.org/[OSGi] framework for registering the extension within Stargate.
```

Reply:
> an OSGI framework 

English grammar is annoying

https://www.merriam-webster.com/words-at-play/is-it-a-or-an